src/main/java/org/mitre/synthea/modules/covid/C19Vaccine.java

```diff
@@ -0,0 +1,122 @@
+package org.mitre.synthea.modules.covid;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.math3.distribution.EnumeratedDistribution;
+import org.apache.commons.math3.util.Pair;
+import org.mitre.synthea.helpers.Utilities;
+import org.mitre.synthea.world.agents.Person;
+
+/**
+ * This class provides information on the currently Emergency Use Authorized (EUA) vaccines in the
+ * United States. It also provides functionality to select a vaccine based on national usage
+ * statistics.
+ * <p>
+ * Note that the Pfizer vaccine has received full approval from the FDA, so technically it is no
+ * longer under EUA.
+ * </p>
+ */
+public class C19Vaccine {
+  public static final HashMap<EUASet, C19Vaccine> EUAs = new HashMap();
+  public static EnumeratedDistribution<EUASet> shotSelector;
```

Comment:
You should probably make this shotSelector `private` since you want to ensure the distribution is properly seeded, as in the `selectShot` method below. It doesn't look like you access this anywhere directly, including within unit tests, so that should probably be safe to change.

Reply:
It is called in the main module, after Janssen is approved. I'm leaving it public but putting in a check to call `initialize` if `shotSelector` is `null`.

Reply:
Not to quibble, but no other class in the code _directly_ touches `shotSelector`.

The main module calls `C19Vaccine.selectShot(person)` after Janssen is approved, which is the _public_ method that should be called.

Reply:
Sorry. Misread your first comment. I can update.